sr_utilities_common/README.md

```diff
@@ -126,4 +126,39 @@ An example launch file using this script could look like the one below:
   </node>
 ```
 
-Launching the above file would upload files `example_file_0` and `example_file_1` located in package `example_package` within folder `example_folder_within_package` to the `shadowrobot.example-bucket` bucket on AWS.
\ No newline at end of file
+Launching the above file would upload files `example_file_0` and `example_file_1` located in package `example_package` within folder `example_folder_within_package` to the `shadowrobot.example-bucket` bucket on AWS.
+
+## Conditional Delayed Ros Tool
+
+This script allows to delay the launch of ros launch files and nodes. It checks on the roscore whether the required conditions have been correctly initialized until the timout exceeds.
+
+### Fields
+
+The node **requires** the following parameters:
+- package_name: the package in which the executable is stored
+- executable_name: could be the name of the roslaunch (without the .launch) or the name of the node
```

Comment:
Here maybe you should allow the user to enter the full name and include the extensions as well. As by doing so you will be able to detect the type (roslaunch or rosnode)